src/PolyhedralGeometry/PolyhedralFan/standard_constructions.jl

```diff
@@ -204,7 +204,7 @@ function _check_containment_via_facet_signs(smaller::Vector{Int}, bigger::Vector
   return true
 end
 function _get_maximal_cones_containing_vector(
-  Sigma::_FanLikeType, v::AbstractVector{<:IntegerUnion}
+  Sigma::_FanLikeType, v::AbstractVector{<:Union{Integer,ZZRingElem,QQFieldElem}}
```

Comment:
I do not yet understand why `QQFieldElem` is needed here. Can you explain? Also, maybe write this as union of `integerUnion` and `QQFieldElem`, so that future updates to IntegerUnion affect/arrive at this line.

Reply:
Changed to `RationalUnion`